net-im/signal-desktop-bin/signal-desktop-bin-1.5.2.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+MY_PN="${PN/-bin/}"
+
+inherit gnome2-utils unpacker xdg-utils pax-utils
+
+DESCRIPTION="Signal Desktop"
```

Comment:
Could you elaborate a little bit on what it is?

Reply:
Done